### PR TITLE
Added recent measures to src/MLJ.jl

### LINKED ISSUE #880: New measures `RSquared` and aliases `rsq`, `rsquared` need exporting
Or we should export measures in a more programmatic way.

### PR DESCRIPTION
**Pull request by sayantan1410**
Fix #880 
Sorry, I tried to do it in the optimum way but as I am new to the repository and also this is my first PR to this repository, so I was little bit scared.  Once get a little bit more familiar to the codebase, I wish to come back to it. 
let me know if I made any mistake. 
@ablaom

### CODE REVIEW

File: src/MLJ.jl [hunk 1]
```diff
@@ -98,7 +98,7 @@ export mav, mae, mape, rms, rmsl, rmslp1, rmsp, l1, l2, log_cosh,
     RMS, rmse, RootMeanSquaredError, root_mean_squared_error,
     RootMeanSquaredLogError, RMSL, root_mean_squared_log_error, rmsl, rmsle,
     RootMeanSquaredLogProportionalError, rmsl1, RMSLP,
-    MAPE, MeanAbsoluteProportionalError, log_cosh_loss, LogCosh, LogCoshLoss
+    MAPE, MeanAbsoluteProportionalError, log_cosh_loss, LogCosh, LogCoshLoss, Rsquared, rsq, rsquared
```
Comment (ablaom) [inline src/MLJ.jl:101]: This line is a bit long. You could start a new one. We try to to follow [BlueStyle](https://github.com/invenia/BlueStyle).

**Review by ablaom — CHANGES_REQUESTED** [review]
@sayantan1410 Thank you very much for this. 

Yes, that's a good idea to make a simple PR first.

Just the minor formatting comment and othewise good.

**Comment by sayantan1410** [comment]
Sure I will make the change ASAP 
Thank you @ablaom

**Comment by sayantan1410** [comment]
@ablaom just mentioning you here, incase you haven't noticed the changes.

**Comment by ablaom** [comment]
Thanks for patience. Just back from leave.

**Comment by sayantan1410** [comment]
That's all right, hope you had a great leave and thanks for merging the PR.

**Pull request merged**